src/proc/proc.cpp

```diff
@@ -178,18 +178,10 @@ int write_contract_args(const contract_exec_args &args)
 
 int feed_inputs(const contract_exec_args &args)
 {
-    // Write any hp input messages to hp->sc pipe.
-    if (write_contract_hp_inputs(args) != 0)
-    {
-        LOG_ERR << "Failed to write HP input to contract.";
-        return -1;
-    }
-
-    // Write any npl inputs to npl pipes.
-    if (write_contract_fdmap_inputs(nplfds, args.nplbufs) != 0)
+    // Write any hp or npl input messages to hp->sc and npl->sc pipe.
+    if (write_contract_hp_npl_inputs(args) != 0)
     {
-        cleanup_fdmap(nplfds);
-        LOG_ERR << "Failed to write NPL inputs to contract.";
+        LOG_ERR << "Failed to write HP or NPL inputs to contract.";
```

Comment:
Error message not needed. `write_contract_hp_npl_inputs` will log the correct error anyway.